src/server-middleware/api/dailyEntries.js

```diff
@@ -23,23 +24,25 @@ export const entriesOfTheDay = (type, config = {}) => {
     key = TOPICS_CACHE_KEY;
   } else if (type === 'item') {
     key = ITEMS_CACHE_KEY;
+  } else if (type === 'itemCountsMediaType') {
+    key = ITEM_COUNTS_BY_MEDIA_TYPE_CACHE_KEY;
   }
 
   return redisClient.getAsync(key)
     .then(value => JSON.parse(value))
     .then(parsed => {
       redisClient.quitAsync();
-      const offset = offsetOfTheDay(parsed.length);
-      return parsed.slice(offset, offset + subsetSize);
+      const offset = offsetOfTheDay(parsed.length, options);
```

Comment:
We do not need the 'random' offset for `itemCountsMediaType`. The offset for type `itemCountsMediaType` is always 0 and the size always 5. 
Could we define `subsetSize` and `offset` conditionally if `type === itemCountsMediaType`? Then we don't have to pass the `options` and remove the condition in the return below.